README.md

```diff
@@ -20,64 +20,65 @@ limitations under the License.
 
-1. Clone and build this repository
+1. Install and run Accumulo via the instructions found in [INSTALL.md] of Accumulo's tarball. 
+   Remember the instance name. It will be referred to as "instance" throughout the examples. A
+   comma-separated list of zookeeper servers will be referred to as "zookeepers".
+
+2. Create an Accumulo user (for help see the 'User Administration' section of the 
+   [user manual][manual]), or use the root user. This user and their password should replace any
+   reference to "username" or "password" in the examples. This user needs the ability to create
+   tables.
+   
+3. Clone and build this repository.
 
         git clone https://github.com/apache/accumulo-examples.git
         mvn clean package
-
-2. Install and run Accumulo via the instructions found in INSTALL.md.
-   Remember the instance name. It will be referred to as "instance" throughout
-   the examples. A comma-separated list of zookeeper servers will be referred
-   to as "zookeepers".
-
-3. Create an Accumulo user (for help see the 'User Administration' section of the 
-   [user manual][manual]), or use the root user. This user and their password
-   should replace any reference to "username" or "password" in the examples. This
-   user needs the ability to create tables.
-
-In all commands, you will need to replace "instance", "zookeepers",
-"username", and "password" with the values you set for your Accumulo instance.
-
-Commands intended to be run in bash are prefixed by '$'. These are always
-assumed to be run the from the root of this repository.
-
-It is expected that the `accumulo` and `tool.sh` commands are your `PATH`.
-These commands are found in the `bin/` and `contrib/` directories of your
-Accumulo installation.
-
-Commands intended to be run in the Accumulo shell are prefixed by '>'.
+          
+4. Each Accumulo example has its own documentation and instructions for running the example. This
+   documentation is linked to below in 'Available Examples'.
+   
+When running the examples, remember the tips below:
+
+* Example is run using the `runex` command which is located in the `bin/` directory of this repo.
+  The `runex` command is a simple wrapper around the Maven Exec plugin.
+* Any command that references Accumulo settings such as `instance`, `zookeepers`, `username`, or 
+`password` should be updated for your instance.
+* Commands intended to be run in bash are prefixed by '$' and should be run from the root of this
+  repository.
+* Several examples use the `accumulo` and `tool.sh` commands which are expected to be on your 
+  `PATH`. These commands are found in the `bin/` and `contrib/` directories of your Accumulo
+  installation.
+* Commands intended to be run in the Accumulo shell are prefixed by '>'.
 
 ## Available Examples
 
 Each example below highlights a feature of Apache Accumulo.
 
-| Example | Description |
-|---------|-------------|
-| [batch] | Using the batch writer and batch scanner |
```

Comment:
I wasn't sure until I looked at both.  Once you put up the new commit, it was easy for me to compare the new and old commit.  When comparing I liked the way the table looked.